src/features/pages/options/vanilla/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { vanillaFAQ, vanillaMarkets, vanillaOptionsContentItems, vanillaPlatforms } from './data'
+import { why_vanilla_section } from './styles.module.scss'
+import AvailableMarketPlatforms from 'features/components/organisms/available-markets-platforms'
+import OptionsContent from 'features/components/templates/options-content'
+import OptionsFaq from 'features/components/templates/options-faq'
+import OptionsLayout from 'features/components/templates/options-layout'
+import Flex from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+
+const VanillaOptions = () => {
+    return (
+        <OptionsLayout
+            heading="_t_What are vanilla options?_t_"
+            description="_t_Vanilla options allow you to express a bullish or bearish view on an underlying asset by purchasing either a Call or a Put option. You can earn a potentially high payout if your predictions are right within a timed contract and based on market conditions._t_"
+        >
+            <Typography.Heading align="center" padding_block="20x" size="small">
+                <Localize translate_text="_t_How do vanilla options work?_t_" />
+            </Typography.Heading>
+            <OptionsContent items={vanillaOptionsContentItems} />
+            <AvailableMarketPlatforms markets={vanillaMarkets} platforms={vanillaPlatforms} />
+            <Flex.Box direction="col" justify="center" gap="16x" className={why_vanilla_section}>
+                <Typography.Heading size="xxs">
+                    <Localize translate_text="_t_Why traders choose vanilla options_t_" />
```

Comment:
Do we have a question mark here?
```suggestion
                    <Localize translate_text="_t_Why traders choose vanilla options_t_" />
```

Reply:
yeah man, will fix it :D

Reply:
oh no we don't need the question mark there man, check the designs.